### DISCUSSION TITLE
Week 4 Report

### DISCUSSION THREAD

**Discussion by derrrobin** [SE-TINF22B6/CookPit#25] [General]
**This week we have been thinking a lot about how the user will best use CooKPit. For this, we created sequence, activity, and use case diagrams to give a better idea of how we want to build the user experience.**

This week we also started talking about how exactly we want to integrate OpenAI. You can read more about this in our [SRS document](https://github.com/SE-TINF22B6/CookPit/blob/main/Docs/srs.md).

Additionally, a important task for this week was the different diagrams. In the following link you can find the diagrams that we generated four our project:
https://github.com/SE-TINF22B6/CookPit/tree/main/Docs/SRS%20Images

We want to use OpenAI to create recipe suggestions for the user with Ki.

This week we also started to think about how a user can upload a recipe. There will be a more detailed report on this in the future. So stay tuned!

We want to offer the user the possibility to upload recipes easily and quickly through the editor and have come up with something special for this.

So stay tuned, your CookPit Team.

**Comment by BenjaminB109** [comment]
Dear CookPit Team,
i think it is a very good idea to use OpenAI for the suggestions of recipes. Also the function to upload recipes would be a very good extension. Keep it up!

**Comment by Maximon132** [comment]
Hey all,
I really like how you use sequence, activity, and use case diagrams to improve the user experience. I'm also interested to learn more about how you plan to integrate OpenAI to create recipe suggestions for users with KI.
Keep up the great work!
